src/components/Layout/SymptomsCarousel/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import SymptomsCards from '../SymptomsCards'
+import './SymptomsCarousel.scss';
+import '../../../styles/_colors.scss';
```

Comment:
You can avoid this import here because this file is imported in the Layout component.

Reply:
What imports should I avoid? all for of them in SymptomsCarousel/index.js ? or just de scss ones?

Reply:
Just the `_colors.scss`